### PR TITLE
#83/improvements

### PR DESCRIPTION
**Pull request by ShravanSunder**
- #83

### CODE REVIEW

File: packages/vite-app-ts/src/MainPage.tsx [hunk 1]
```diff
@@ -99,44 +97,47 @@ export const Main: FC = () => {
     setRoute(window.location.pathname);
   }, [setRoute]);
 
+  // -----------------------------
+  // 📃 Page List
+  // -----------------------------
+  // This is the list of pages and tabs
+  const pageList: TContractPageList = {
+    mainPage: {
```
Comment (grothem) [inline packages/vite-app-ts/src/MainPage.tsx:105]: Not a huge deal, but could also add the new YourToken contract here

File: packages/vite-app-ts/src/components/main/createPagesAndTabs.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+};
+
+/**
+ * Helper function that creates pages with routes.  It also creates tabs (menu items) associated with those routes, so that you can click on them to navigate to the page.
+ * @param pageList
+ * @param route
+ * @param setRoute
+ * @returns
+ */
+export const createPagesAndTabs = (
```
Comment (grothem) [inline packages/vite-app-ts/src/components/main/createPagesAndTabs.tsx:30]: Nice helper for creating the routes!

**Review by grothem — COMMENTED** [review]
Overall looks like some good improvements. I like scaffold-common project for sharing stuff between the front end and hardhat.

I did noticed you removed ExampleUI.tsx, any particular reason? Could be helpful for new people just getting started.

I pulled your branch and fired everything up, I didn't have any issues!

**Comment by ShravanSunder** [comment]
> Overall looks like some good improvements. I like scaffold-common project for sharing stuff between the front end and hardhat.
> 
> I did noticed you removed ExampleUI.tsx, any particular reason? Could be helpful for new people just getting started.
> 
> I pulled your branch and fired everything up, I didn't have any issues!

@grothem 
i mostly removed it as it was a lot of overhead keeping it working and not much value.  People can just use the eth-components library now and look at the docs for understanding UX components

File: packages/vite-app-ts/src/MainPage.tsx (hunk 1 above)
Comment (ShravanSunder) [reply packages/vite-app-ts/src/MainPage.tsx:105]: added!

**Review by grothem — APPROVED** [review]

**Pull request merged**